packages/orchestrator-ui-components/src/components/WfoTable/WfoAdvancedTable/getRowDetailData.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import {
+    WfoAdvancedTableColumnConfig,
+    WfoAdvancedTableDataColumnConfigItem,
+} from '@/components/WfoTable/WfoAdvancedTable/types';
+import {
+    ColumnType,
+    WfoTableControlColumnConfigItem,
+} from '@/components/WfoTable/WfoTable';
+import { getTypedFieldFromObject } from '@/utils';
+
+export const getRowDetailData = <T extends object>(
+    selectedDataForDetailModal: T | undefined,
+    tableColumnConfig: WfoAdvancedTableColumnConfig<T>,
+) => {
+    if (!selectedDataForDetailModal) {
+        return undefined;
+    }
+
+    const tableColumnConfigEntries: [
+        string,
+        (
+            | WfoTableControlColumnConfigItem<T>
+            | WfoAdvancedTableDataColumnConfigItem<T, keyof T>
+        ),
+    ][] = Object.entries(tableColumnConfig);
+
+    const dataColumnEntries = tableColumnConfigEntries.filter(
+        (tableColumnConfigEntry) =>
```

Comment:
I prefer to do something more descriptive like this so you don't have to do the access by [1] in line 31
```suggestion
        ([_, columnConfig]) =>
```

Reply:
I prefer this too, but the current linting rules do not allow this. It still sees `_` as an unused variable